### ISSUE TITLE
Fixing accessibility markup for all image elements and image like elements

### ISSUE THREAD

**Issue by winniebosy** [EddieHubCommunity/good-first-issue-finder#279]
### Description

I have noticed that some of the images as well as image-like elements(SVGs) are missing descriptive texts while some that are considerably decorative are still accessed by assistive technologies. This can be confusing to users with visual disabilities because they will fail to understand the content or purpose of the images. 

I am requesting to fix all the accessibility issues relating to the site's images/graphics.

### Screenshots

![image](https://user-images.githubusercontent.com/40880690/214526295-ba5309b3-d0f6-48f4-8a05-c7fea7154f67.png)

**Comment by alokVerma749** [comment]
I think we can provide descriptive text for SVG's as:
```
<svg role="img" aria-label="A red heart icon">
  <title>A red heart icon</title>
  <path d="M12,1.8c-4.7,0-8.6,3.8-8.6,8.6c0,7.9,8.6,15.6,8.6,15.6s8.6-7.7,8.6-15.6"/>
</svg>
```

**Comment by Cahllagerfeld** [comment]
awesome, do you want to work on this then? :)

**Comment by alokVerma749** [comment]
ya sure, i can work on this, you can assign me

**Comment by alokVerma749** [comment]
Please review the changes
